### PR TITLE
Handle gap in prescribe.py example

### LINKED ISSUE #98: Update prescriptor example to fill any case data gap
The current neat example assumes case data is available until start_date. This is not true in general, so the predictor should be used in prescribe.py to fill the missing data.

### PR DESCRIPTION
**Pull request by EKMeyerson**
resolves #98

### CODE REVIEW

File: covid_xprize/examples/prescriptors/neat/prescribe.py [hunk 1]
```diff
@@ -6,23 +6,35 @@
 import pandas as pd
 
 from copy import deepcopy
+from datetime import datetime
 
 import neat
```
Comment (EKMeyerson) [inline covid_xprize/examples/prescriptors/neat/prescribe.py:12]: Split out imports into single lines for readability and ease of tracking

Comment (EKMeyerson) [reply covid_xprize/examples/prescriptors/neat/prescribe.py:12]: `load_ips_file` is the only new import in this block

File: covid_xprize/examples/prescriptors/neat/prescribe.py [hunk 2]
```diff
@@ -32,25 +44,54 @@ def prescribe(start_date_str: str,
     start_date = pd.to_datetime(start_date_str, format='%Y-%m-%d')
     end_date = pd.to_datetime(end_date_str, format='%Y-%m-%d')
```
Comment (EKMeyerson) [inline covid_xprize/examples/prescriptors/neat/prescribe.py:45]: Previously, this file was ignoring the `prior_ips_file`, and simply pulling the ips from the historical data file. Now we actually use the `prior_ips_file`.

File: covid_xprize/examples/prescriptors/neat/prescribe.py [hunk 3]
```diff
@@ -32,25 +44,54 @@ def prescribe(start_date_str: str,
...
 
+    # Load the past IPs data
+    print("Loading past IPs data...")
+    past_ips_df = load_ips_file(path_to_prior_ips_file)
```
Comment (EKMeyerson) [inline covid_xprize/examples/prescriptors/neat/prescribe.py:49]: The `geos` we prescribe for are defined by the `prior_ips_file`

File: covid_xprize/examples/prescriptors/neat/prescribe.py [hunk 4]
```diff
@@ -128,15 +171,13 @@ def prescribe(start_date_str: str,
             pred_df = get_predictions(start_date_str, date_str, pres_df)
 
             # Update past data with new day of prescriptions and predictions
```
Comment (EKMeyerson) [inline covid_xprize/examples/prescriptors/neat/prescribe.py:173]: standardizing how `GeoID` is created in this file

File: covid_xprize/examples/prescriptors/neat/prescribe.py [hunk 5]
```diff
@@ -155,9 +196,12 @@ def prescribe(start_date_str: str,
 
     # Combine dfs for all prescriptions into a single df for the submission
     prescription_df = pd.concat(prescription_dfs)
+    prescription_df = prescription_df.drop(columns='GeoID')
```
Comment (EKMeyerson) [inline covid_xprize/examples/prescriptors/neat/prescribe.py:200]: Make the code more flexible. I think this actually fixes #117 .

File: covid_xprize/examples/prescriptors/neat/utils.py [hunk 6]
```diff
@@ -68,7 +68,6 @@ def prepare_historical_df():
                   parse_dates=['Date'],
                   encoding="ISO-8859-1",
                   error_bad_lines=False)
```
Comment (EKMeyerson) [inline covid_xprize/examples/prescriptors/neat/utils.py:70]: part of standardizing how `GeoID` is created

File: covid_xprize/validation/data/fixed_equal_costs.csv [hunk 7]
```diff
@@ -9,6 +9,7 @@ Aruba,,1,1,1,1,1,1,1,1,1,1,1,1
 Australia,,1,1,1,1,1,1,1,1,1,1,1,1
```
Comment (EKMeyerson) [inline covid_xprize/validation/data/fixed_equal_costs.csv:9]: Update example cost files with the new regions

File: covid_xprize/examples/prescriptors/neat/utils.py [hunk 8]
```diff
@@ -87,6 +86,15 @@ def prepare_historical_df():
     return df
 
 
+# Function to load an IPs file, e.g., passed to prescribe.py
+def load_ips_file(path_to_ips_file):
```
Comment (ofrancon) [inline covid_xprize/examples/prescriptors/neat/utils.py:90]: Could use `load_dataset` from `predictor_scoring.py` to also filter out the geos we're not considering

**Review by ofrancon — APPROVED** [review]
lgtm, thanks!

**Review by ofrancon — APPROVED** [review]

**Pull request merged**